### PR TITLE
 Create art view page, add styling, add pull requst template

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Create art view page, add styling, and add pull request template.

What (if anything) did you refactor?
- Nothing

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+What (if any) features are you implementing?
```
Comment (StevieAmb) [inline pull_request_template.md:1]: Thank you for making this PR template.

File: src/ArtView/ArtView.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
+.art-view-container {
```
Comment (StevieAmb) [inline src/ArtView/ArtView.scss:1]: Great work on this CSS

File: src/ArtView/ArtView.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+.user-choices {
+  display: flex;
+  justify-content: center;
+}
+
+.featured-art {
+  max-width: 95%;
+}
+
+.ratings-choice {
```
Comment (StevieAmb) [inline src/ArtView/ArtView.scss:17]: This looks great!

File: src/ArtView/ArtView.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import './ArtView.scss';
+
+interface State {
+  userLikesPhoto: boolean | undefined;
+}
+
+interface Props {
```
Comment (StevieAmb) [inline src/ArtView/ArtView.tsx:8]: Great work with this interface!

**Review by StevieAmb — APPROVED** [review]

File: src/ArtView/ArtView.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+      <div className="art-view-container">
+           <img className="featured-art" src="https://www.artic.edu/iiif/2/1adf2696-8489-499b-cad2-821d7fde4b33/full/843,/0/default.jpg" alt="painting of people at a park on a sunny day" />
+        <div className="user-choices">
+          <button className="ratings-choice" >👍</button>  
+          <button className="ratings-choice" >👎</button>  
+        </div>
+      </div>
+    )
+  }
+}
```
Comment (mr-ryan12) [inline src/ArtView/ArtView.tsx:37]: Awesome implementation of using a class component. We'll need to hold some state definitely.

File: src/Categories/Categories.scss [hunk 6]
```diff
@@ -6,3 +6,9 @@
   text-align: center;
   padding: 25px;
 }
+
+@media screen and (min-device-width: 0px) and (max-device-width: 800px) {
+  .categories-container {
+    padding: 15px;
+  }
+}
```
Comment (mr-ryan12) [inline src/Categories/Categories.scss:14]: Thank you for keeping mobile user's in consideration with the design!

**Pull request merged**